src/sign-verify/clu_verify.c

```diff
@@ -19,12 +19,10 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
  */
 
-#include "clu_include/sign-verify/clu_verify.h"
-#include "clu_include/sign-verify/clu_sign.h" /* for RSA_SIG_VER, ECC_SIG_VER, 
-                                                      ED25519_SIG_VER */
-#include <wolfssl/wolfcrypt/types.h>
-#include <wolfssl/ssl.h>
-#include "clu_include/clu_header_main.h"
+#include <wolfclu/clu_header_main.h>
+#include <wolfclu/sign-verify/clu_verify.h>
+#include <wolfclu/sign-verify/clu_sign.h> /* for RSA_SIG_VER, ECC_SIG_VER,
```

Comment:
Future improvement: I don't think this comment is necessary. We're not commenting elsewhere why we've included a given header file. Keep it simple, IMO.